### PR TITLE
Feature/1202839432398962 implement transaction auto approval

### PR DESCRIPTION
**Pull request by irediaes**
* added transaction sqlite db for pending approval storage
* saved transaction info on service delivered event
* added auto approval countdown before triggering transaction approval
* added auto approval trigger when consumer click the back button without authorizing transaction
* added auto approval of pending transactions on app initialization

### CODE REVIEW

File: lib/common/providers/charge_provider.dart [hunk 1]
```diff
@@ -264,15 +331,45 @@ class CEVChargeProvider with ChangeNotifier {
       return;
     }
 
-    setStatus(LoadingStatus.loading);
+    var otherSignatories = [_station.address!];
```
Comment (sfffaaa) [inline lib/common/providers/charge_provider.dart:334]: Do we deal with the below scenario in this modification? Or in our requirement, we don't need to worry about this.

1. UserA connects to stationA, then the extrinsic of sending/refunding is pending.
2. UserA connects to stationB, and shows the pending extrinsic for the stationA?

File: lib/config/env.dart [hunk 2]
```diff
@@ -75,8 +76,8 @@ class Env {
   static const String extrinsicSuccessEvent = "ExtrinsicSuccess";
   static const String extrinsicFailedEvent = "ExtrinsicFailed";
   static const String approved = "Approved";
-  static const String peaqTestnet = "wss://wss.agung.peaq.network";
-  // static const String peaqTestnet = "ws://10.0.2.2:9944";
+  // static const String peaqTestnet = "wss://wss.agung.peaq.network";
+  static const String peaqTestnet = "ws://10.0.2.2:9944";
```
Comment (sfffaaa) [inline lib/config/env.dart]: Should we change to the Agung network?

Comment (irediaes) [reply lib/config/env.dart]: Sorry, I forgot to change it back to production url. 
Thanks for the catch.

File: lib/common/providers/charge_provider.dart (hunk 1 above)
Comment (irediaes) [reply lib/common/providers/charge_provider.dart:334]: There's no separation of StationA or B. We are just saving the transaction received from ServiceDelivered event. 
1. We save the transaction to local db.
2. We fetch the transaction from db on next app opening if the app was exited without approving transaction.
3. We delete transaction from local db only after successful approval.

File: lib/config/env.dart (hunk 2 above)
Comment (irediaes) [reply lib/config/env.dart]: Fixed

**Review by sfffaaa — APPROVED** [review]

**Pull request merged**